spec/unit/lib/cloud_controller/metrics/prometheus_updater_spec.rb

```diff
@@ -1,46 +1,12 @@
 require 'spec_helper'
 require 'cloud_controller/metrics/prometheus_updater'
 
-# There are a few things to NOTE here:
-# 1) We're adding this function because the Prometheus Client Registry
-#    is interpreter-global state, so changes that happen in one test will
-#    absolutely affect other tests, unless the test author has taken pains
-#    to carefully clean up their changes.
-# 2) In our CI, tests are run in parallel to speed up execution. At the time
-#    of this writing (2022-05-03), we use the `grosser/parallel_tests` gem to
-#    run tests in parallel. This gem uses the `grosser/parallel` gem to distribute
-#    work which runs workloads in separate subprocesses, rather than threads
-#    in the same interpreter process.
-#    You will notice that we call this function that we created in a top-level
-#    `before` block a little bit later in this file, and _also_ that we mention
-#    above that the Registry is interpeter-global state. Because our
-#    test-parallelizer uses subprocesses to distribute its work, and also does
-#    not interleave tests from one file with tests from another, the registry
-#    recreation that we're doing here will be restricted to the tests in _this_
-#    file and will not leak out into other files.
-# 3) So, if you see weird or unreliable behavior out of the Registry when running
-#    specs (this would probably be stuff like "metric values are sometimes not what
-#    they should be"), check to see if our test parallelizer is still using
-#    subprocesses, or if it has switched to threads and runs everything in one
-#    interpreter.
-module Prometheus
-  module Client
-    def self.recreate_registry
-      @registry = Prometheus::Client::Registry.new
-    end
-  end
-end
-
 module VCAP::CloudController::Metrics
   RSpec.describe PrometheusUpdater do
     let(:updater) { PrometheusUpdater.new(prom_client) }
-    let(:prom_client) { Prometheus::Client.registry }
+    let(:prom_client) { Prometheus::Client::Registry.new }
```

Comment:
Am I missing something, or does this do the same job as the monkey patch, I deleted above?